composer.json

```diff
@@ -15,8 +15,8 @@
         "psr-4": {"OckCyp\\CoversValidator\\Tests\\": "tests/"}
     },
     "require": {
-        "php": "^7.0",
-        "phpunit/phpunit": "^6.0 || ^7.0",
+        "php": "^7.1",
```

Comment:
do we need to change php version constraint since it's only a test that breaks compatibility? if we don't then everyone will be able to use 1.1.0

if we're actually dropping 7.0 support I'd probably use return types in the project and then we could do a major release

Reply:
I was also thinking about this but figured it is unlikely to make much of a difference, especially considering there are no changes between this version and the previous one, and it does not seem likely there will be important ones any time soon.

Personally I'd also avoid making a major release for return types. For users, the cost of needing to care about the version of this package is  higher than any benefit they get from return types (basically none, since this is a tool and not a lib).

Reply:
I was thinking more to keep only one latest version. I'll avoid making a major release